st2web/files/st2.conf-http.patch

```diff
@@ -1,18 +1,20 @@
---- /etc/nginx/conf.d/st2.template	2021-04-01 19:37:46.852016859 +0200
-+++ /etc/nginx/conf.d/st2.conf	2021-04-01 19:39:30.233113482 +0200
-@@ -9,47 +9,12 @@
+--- st2.conf	2021-06-18 19:45:50.892196136 +0100
++++ st2.http.conf	2021-06-18 19:50:00.098280995 +0100
+@@ -8,44 +8,18 @@
+ 
  server {
    listen *:80 default_server;
++  server_tokens off;
```

Comment:
Wasn't sure about this line
In the version in the package, server_tokens off is only in https not http section?

Reply:
In a normal install, HTTP is redirected to HTTPS and so this setting goes to HTTPS section.

In Docker environment, we split HTTP vs HTTPS configs and so server token needs to be set for the config that's enabled.
In this case `st2.conf-http.patch`.